spec/features/owner_actions_spec.rb

```diff
@@ -253,7 +253,7 @@
     page.find('a', text: 'Your Profile').click
     expect(page).to have_content(@owner.display_name)
     expect(page).to have_selector('.columns')
-    expect(page).not_to have_content("Recent Activity by #{@owner.display_name}")
+    expect(page).to have_content("Recent Activity by #{@owner.display_name}")
```

Comment:
Good question -- I'm not actually sure what the answer is off the top of my head.

Reply:
Looks like we added it when we switched to friendly ids and (more) restful routes and URLs, instead of the old `?collection_id=3&action=show` style we started with back in Rails 1.2.